.github/workflows/build.yml

```diff
@@ -59,9 +59,6 @@ jobs:
                 cmakeVersion: 3.22.1
             - run: sudo apt update
             - run: sudo apt install --yes --no-install-recommends libwayland-dev libxrandr-dev
-            # This is to combat a bug when using 6.6 linux kernels with thread/address sanitizer
-            # https://github.com/google/sanitizers/issues/1716
-            - run: sudo sysctl vm.mmap_rnd_bits=28
```

Comment:
Definitely safe to remove - I love removing workarounds for bugs that no longer affect us.